### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by RahullRawat**
Screenshot (152)

### CODE REVIEW

File: src/pages/Landing/Landing.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import heroImg from "../../assets/hero-img.png";
+import noteIcon from "../../assets/note-icon.png";
+
+const Landing = () => {
+	return (
+		<div className="landing-container">
+			<header className="header">
+				<a href="#" className="header-container">
+					<h4 className="xl-text">TopNote </h4>
+					<img src={noteIcon} alt="" className="note-logo" />
```
Comment (Ayushi20-19) [inline src/pages/Landing/Landing.js]: provide proper alt from images

**Review by Ayushi20-19 — APPROVED** [review]
Good naming convention
Clean file structure

**Review by NamrataRaikwar2002 — APPROVED** [review]
Clean code, nice UI, great work Rahul.

**Review by AtulPant2704 — APPROVED** [review]
Everything looks good, clean code and file structure.

**Pull request merged**